File: YAzyki programmirovaniya/5 zadach' po fizike na 7 yazykakh programmirovaniya/Задачи C#/Задачи/Form5.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Задачи
{
    public partial class Form5 : Form
    {
        public Form5()
        {
            InitializeComponent();
        }
        double S1, S2, t, v1, v2;

        private void button1_Click(object sender, EventArgs e)
        {
            t = Convert.ToDouble(textBox2.Text);
            v1 = Convert.ToDouble(textBox1.Text);
            v2 = Convert.ToDouble(textBox4.Text);
            S1 = v1 * t;
            S2 = v2 * t;

            textBox3.Text = Convert.ToString(S1);
            textBox5.Text = Convert.ToString(S2);
        }

        private void Form5_Load(object sender, EventArgs e)
        {
            
        }
    }
}

File: YAzyki programmirovaniya/5 zadach' po fizike na 7 yazykakh programmirovaniya/Задачи VB/WindowsApplication1/Form5.vb
Public Class Form5
    Dim S1, S2, t, v1, v2 As Double
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        t = Val(textBox2.Text)
        v1 = Val(textBox1.Text)
        v2 = Val(textBox4.Text)

        S1 = v1 * t
        S2 = v2 * t

        textBox3.Text = CStr(S1)
        textBox5.Text = CStr(S2)
    End Sub

    Private Sub Form5_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub
End Class